module/handlebar-helpers.mjs

```diff
@@ -56,4 +56,15 @@ export default function registerHandlebarHelpers() {
   Handlebars.registerHelper( 'ed-linkForUuid', linkForUuid );
 
   Handlebars.registerHelper( 'ed-diceFormulaForStep', getDice );
+
+  Handlebars.registerHelper( "getTalentType", ( talents, type ) => {
+    return talents.filter( ( talent ) => talent.system.talentType === type );
+  } );  
+
+  Handlebars.registerHelper( "getKnackName", ( talent, knacks ) => {
```

Comment:
StringCompare not good. That's why we use the ForeignDocument, or the UUIDFields.